src/services/ClientService.ts

```diff
@@ -170,18 +179,18 @@ class ClientService {
   ];
 
   async newRequest(
-    request: ServiceRequestType,
-    client: ClientType,
-    animal: AnimalType,
+    request: EditableServiceRequestType,
+    client: EditableClientType,
+    animal: EditableAnimalType,
   )
     : Promise<ServiceRequestType> {
     // Post Request to supabase
 
     let createdRequest: ServiceRequestType;
 
     try {
-      let animalId: BigInteger;
```

Comment:
why did you switch to string here?

Reply:
To match the current type of the ids in the schema.